src/utils/issue.ts

```diff
@@ -0,0 +1,171 @@
+import { Context } from "../types/context";
+import { Issue, ISSUE_TYPE } from "../types/payload";
+import { getLinkedPullRequests } from "./get-linked-prs";
+
+export function isParentIssue(body: string) {
+  const parentPattern = /-\s+\[( |x)\]\s+#\d+/;
+  return body.match(parentPattern);
+}
+
+export async function getAssignedIssues(context: Context, username: string): Promise<Issue[]> {
+  const payload = context.payload;
+
+  try {
+    return await context.octokit.paginate(
+      context.octokit.issues.listForRepo,
+      {
+        owner: payload.repository.owner.login,
+        repo: payload.repository.name,
+        state: ISSUE_TYPE.OPEN,
+        per_page: 1000,
```

Comment:
I wonder if `1000` is the correct number. Why did you use this?

Reply:
Must have been a typo as I know 100 is the max, surprised it didn't error but it likely defaults to max